static/scss/style.scss

```diff
@@ -18,8 +18,7 @@ $viewport-md: 768px;
 $viewport-lg: 992px;
 $viewport-xl: 1200px;
 
-@import "../vendor/bootstrap@4.5.3/scss/bootstrap.scss";
-
+@import "~bootstrap/scss/bootstrap";
```

Comment:
For the records, this makes the build fail because we're still running the old check sass step in CI. We should probably replace this step with one with our new npm build.

Reply:
Yes currently working on this